modules/db/src/main/java/org/mule/module/db/internal/domain/param/DefaultInputQueryParam.java

```diff
@@ -14,20 +14,38 @@
 
     private final Object value;
 
+    private final boolean isDbParam;
+
     public DefaultInputQueryParam(int index, DbType type, Object value)
     {
-        this(index, type, value, null);
+        this(index, type, value, null, false);
     }
 
     public DefaultInputQueryParam(int index, DbType type, Object value, String name)
     {
+        this(index, type, value, name, false);
+    }
+
+    public DefaultInputQueryParam(int index, DbType type, Object value, String name, boolean isDbParam)
```

Comment:
But I used this contructor to set the property with the value "true" in SimpleQueryTemplateParser